db/views/analytics_client_files_v01.sql

```diff
@@ -0,0 +1,27 @@
+with file_tags as (
+  select taggings.taggable_id as file_id,
+    array_agg(tags.name) as tag_names
```

Comment:
Maybe include order and distinct to ensure consistency?  I think the syntax is something like this:
```suggestion
    array_agg(DISTINCT tags.name ORDER BY tags.name) as tag_names
```